<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orders</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ordernumber</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>animalsid</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>customersid</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>goodsid</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>date</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/search.png</normaloff>pic/search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>570</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search by id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineOrd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search by ordernum</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAnim">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search by animid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCust">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search by custid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineGood">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search by goodsid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search by date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/search.png</normaloff>pic/search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/search.png</normaloff>pic/search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/search.png</normaloff>pic/search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/search.png</normaloff>pic/search.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>590</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 0, 255);
background-color: rgb(233, 205, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/search.png</normaloff>pic/search.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
